### PR TITLE
dummy_RAW

### PR DESCRIPTION
**Pull request by Jacfomg**
Add RAW acquisition to dummy.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Comment by AleCandido** [comment]
@Jacfomg is this ready for review? Otherwise could you make it draft?

**Comment by Jacfomg** [comment]
Sorry, it was ready for main, then @rodolfocarobene told me about him changing it as well and decided to point it to his. Now I will try to fix the problems.

**Review by rodolfocarobene — APPROVED** [review]

**Comment by aorgazf** [comment]
Hi @Jacfomg,
Thanks for taking care of this.
Could you please add a description of what this PR does. This would be beneficial for other members of the team that may join in the future and are not familiar with the code yet.

File: src/qibolab/instruments/dummy.py [hunk 1]
```diff
@@ -73,7 +77,10 @@ def sweep(
         for ro_pulse in sequence.ro_pulses:
             if options.acquisition_type is AcquisitionType.DISCRIMINATION:
                 values = np.random.rand(expts)
-            else:
+            if options.acquisition_type is AcquisitionType.RAW:
+                samples = int(ro_pulse.duration * self.sampling_rate)
+                values = np.random.rand(samples * expts) * 100 + 1j * np.random.rand(samples * expts) * 100
+            if options.acquisition_type is AcquisitionType.INTEGRATION:
```
Comment (AleCandido) [inline src/qibolab/instruments/dummy.py:83]: Same here:
```suggestion
            elif options.acquisition_type is AcquisitionType.INTEGRATION:
```

File: src/qibolab/instruments/dummy.py [hunk 2]
```diff
@@ -42,14 +44,16 @@ def disconnect(self):
...
-        ro_pulses = {pulse.qubit: pulse.serial for pulse in sequence.ro_pulses}
-
         expts = 1 if options.averaging_mode is AveragingMode.CYCLIC else options.nshots
+
         results = {}
         for ro_pulse in sequence.ro_pulses:
             if options.acquisition_type is AcquisitionType.DISCRIMINATION:
                 values = np.random.rand(expts)
-            else:
+            if options.acquisition_type is AcquisitionType.RAW:
```
Comment (AleCandido) [inline src/qibolab/instruments/dummy.py:53]:
```suggestion
            elif options.acquisition_type is AcquisitionType.RAW:
```

File: src/qibolab/instruments/dummy.py [hunk 3]
```diff
@@ -42,14 +44,16 @@ def disconnect(self):
...
+                samples = int(ro_pulse.duration * self.sampling_rate)
+                values = np.random.rand(samples * expts) * 100 + 1j * np.random.rand(samples * expts) * 100
+            if options.acquisition_type is AcquisitionType.INTEGRATION:
```
Comment (AleCandido) [inline src/qibolab/instruments/dummy.py:56]:
```suggestion
            elif options.acquisition_type is AcquisitionType.INTEGRATION:
```

**Review by AleCandido — APPROVED** [review]

**Pull request merged**